enterprise_access/apps/content_assignments/migrations/0012_assignmentaction_add_automatic_cancellation_option.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.2.6 on 2023-12-01 08:50
```

Comment:
Throwing this in here as a separate commit.  It was a missed migration from 7242525a034bcded3d9d59230ebafaac1030401d.

Reply:
FYI @muhammad-ammar